vendor/licenses/grammar/vscode-plantuml.txt

```diff
@@ -0,0 +1,24 @@
+---
+type: grammar
+name: vscode-plantuml
+version: 06ee118c4eb34dd6566a7e648ddb1c3a836a042f
+license: other
```

Comment:
The test failure is because of this line. For some reason the license detector hasn't been able to confirm this is MIT licensed.

Feel free to manually change this:

```suggestion
license: mit
```